integration/eclipse/vite.config.js

```diff
@@ -8,16 +8,7 @@ export default defineConfig(() => {
     plugins: [tsconfigPaths()],
     build: {
       outDir: 'build',
-      chunkSizeWarningLimit: 5000,
-      rollupOptions: {
-        output: {
-          manualChunks: id => {
-            if (id.includes('@axonivy/inscription')) {
-              return 'inscription';
-            }
-          }
-        }
-      }
```

Comment:
somehow rollup breaks something in the naming
It makes out of `new URL('editor.worker-...js')` -> `new rR('editor.worker...js')`
which results in "rR is not a constructor" message